exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h

```diff
@@ -81,7 +81,7 @@ class InMemorySpanExporter final : public opentelemetry::sdk::trace::SpanExporte
   }
 
   /**
-   * @param timeout an optional value containing the timeout of the exporter
+   * Timeout is an optional value containing the timeout of the exporter
```

Comment:
I think `@param timeout` should remain.

Also good to add a small brief: "Attempt to shut down the in-memory span exporter." for the sake of completion.